dspace-api/src/main/java/org/dspace/handle/HandleClarinServiceImpl.java

```diff
@@ -430,6 +430,36 @@ public String getDeadSince(Context context, String handle) throws SQLException {
                 format(timestamptz) : null;
     }
 
+    @Override
+    public Handle createHandle(Context context, String handleStr) throws SQLException, AuthorizeException {
+        // Check authorisation: Only admins may create DC types
+        if (!authorizeService.isAdmin(context)) {
```

Comment:
Only admin? Will never any user use this method during creating an Item? This is only migration purposes right?